Loop/Managers/LoopAppManager.swift

```diff
@@ -43,8 +47,8 @@ class LoopAppManager: NSObject {
         var next: State { State(rawValue: rawValue + 1) ?? .launchComplete }
     }
 
+    private var windowProvider: WindowProvider?
```

Comment:
I think this will need to be `weak` or it will create a circular reference.

Reply:
True, will update.  Also updated `OnboardingManager` with weak reference.